app/src/main/res/layout/question_player_feedback_item.xml

```diff
@@ -0,0 +1,29 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+  </data>
+
+  <FrameLayout
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="28dp"
+    android:layout_marginTop="@dimen/divider_margin_top"
```

Comment:
Margin top is too less. In mock it is 36dp.

Reply:
This will be resolved when content margin  bottom 28dp  get added.